src/components/Offers/Form/form-components/OfferForm.js

```diff
@@ -247,7 +248,7 @@ const OfferForm = ({ context, title }) => {
                                             }}
                                         />
                                     </Grid>
-                                    <Grid item xs={12} lg={12}>
+                                    <Grid item xs={12} lg={6}>
```

Comment:
Adding the following styles to this grid
```css
display: flex;
align-items: center;
```
gives me the following result
![image](https://user-images.githubusercontent.com/13498603/181862314-f4edc619-30a7-4865-87b5-a556a3f44715.png)

Is this what you were looking for?

Reply:
That seems good to me. Do you think the layout looks good like that?

Reply:
Honestly, when the Advanced Settings are open, it looks too cluttered. I was experimenting and I think we could go with something like this

- When Advanced Settings are not shown
![image](https://user-images.githubusercontent.com/13498603/182028403-e436d487-d6f9-4870-acab-c961291182a7.png)

- When Advanced Settings are shown
![image](https://user-images.githubusercontent.com/13498603/182028437-2957b97d-566a-4eef-8b5d-0d26e918f8d0.png)


Optionally, we could also create an entire "section" dedicated to this, such as the Contacts section or the Requirements section, but without the Add Entry button, like so
![image](https://user-images.githubusercontent.com/13498603/182028531-0ecd8904-be4a-4105-8d35-e3251b084a95.png)

Reply:
@BrunoRosendo change this, fix the test and then rebase. Everything should be good to go.

Reply:
@Naapperas Ok boss, will do :+1: